chain/messagepool/selection_test.go

```diff
@@ -1598,7 +1598,7 @@ readLoop:
 
 	mp, tma := makeTestMpool()
 
-	block := tma.nextBlockWithHeight(build.UpgradeBreezeHeight + 10)
+	block := tma.nextBlockWithHeight(uint64(build.UpgradeBreezeHeight + 10))
```

Comment:
I notice you're not changing any of the `build.Upgrade*` references in the code, is there a reason to leave those alone? Just the number of them?

Reply:
Or are you only touching packages that you want to import in Curio?

Reply:
I only brought-over what I needed for Curio for now, but I can keep bringing over these things later as they probably will be needed. I'm trying to unblock a PR for now. Lotus is free to engineer a cleaner API than the minimum I've had to do to get things working.